container-setup/install/install-aws.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+
+if [ "$I_AM_IN_CONTAINER" != "I-am-in-container" ]; then
```

Comment:
What is the benefit of this? To prevent accidental install?  Could this just be a non-executable file that is set to executable when copied into the container instead?  This just adds extra complexity to all the files and the Dockerfile/is not DRY.

Reply:
Ooohhh, I really like that idea.  I'll card that up.

Reply:
Yes, intention is to block execution on your local machine, in case of accidental execution outside of container. +1, I like the "not executable until required".